Makefile

```diff
@@ -70,17 +86,21 @@ $(LINTER_VERSION_FILE):
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s $(GOLANGCI_LINT_VERSION)
 	touch $(LINTER_VERSION_FILE)
 
-lint: $(LINTER_VERSION_FILE)
-	$(LINTER) run ./...
-
+lint: $(LINTER_VERSION_FILE) $(ALL_LINT_TARGETS)
 
 TEMP_TEST_OUTPUT=/tmp/sdk-contract-test-service.log
 
 # TEST_HARNESS_PARAMS can be set to add -skip parameters for any contract tests that cannot yet pass
 TEST_HARNESS_PARAMS=
 
+workspace:
+	rm -f go.work
+	go work init ./
+	go work use ./ldotel
+	go work use ./testservice
```

Comment:
Maybe we could have a workspace-rm or whatever? Or just git reset does the job, I suppose

Reply:
I could add `make clean` back. I could also change this to be a traditional build target, where it makes go.work and clean removes it.

Clean should probably remove the linter download as well.